draft-ietf-scitt-architecture.md

```diff
@@ -619,6 +619,40 @@ For example, consider a Transparency Service implemented using a set of replicas
 Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
 This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
 
+### Configuration
+
+The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/scitt-configuration`.
```

Comment:
```suggestion
The Transparency Service records its configuration in the Append-Only Log using Signed Statements with distinguished media type `application/scitt-configuration`.
```

Reply:
This is an interesting question, whether we reference the Signed Statement before or after it's written. 
...records its configuration... could infer the signed statement is being submitted, becoming a transparent statement.
Is there a strong opinion either way?

Reply:
As usual, what matters for most parties are (signed) transparent statements. In particular, we must reference configuration statements once registered, to ensure they are transparent before they are used for further registrations.

Reply:
Editors meeting discussion: Signed Statement refers to a configuration that doesn't prove it was placed into the transparency service. 
There's a good discussion whether we should use Transparent Statement or Receipt as the term for referencing something that was registered (notarized). However, we should track that separately. 
For this PR, it's important to recognize the configuration was registered, making "signed statement" too early and incorrect.